tests/subsys/storage/stream/stream_flash/src/main.c

```diff
@@ -98,7 +98,7 @@ static void init_target(void)
 	zassert_equal(rc, 0, "expected success");
 }
 
-static void test_stream_flash_init(void)
+ZTEST(lib_stream_flash_test, test_stream_flash_init)
```

Comment:
remove _test as a prefix from testsuite name.

Reply:
done, I have remove the _test